### PR TITLE
Fix Restore sheet gap on Android

### PR DESCRIPTION
**Pull request by jkadamczyk**
## What changed (plus any additional context for devs)

* Changes some code in the RestoreSheet screen
* Made it platform-specific to address the issue

## Screen recordings / screenshots

Before on my Pixel 6
image

After on pixel and iOS:

https://user-images.githubusercontent.com/16062886/195832119-4d53d487-0317-4392-b487-42923e26ce67.mp4

https://user-images.githubusercontent.com/16062886/195832234-1f44eecb-54d7-4ad1-8685-a2cc8386d8e7.mov

## What to test

* Test Restore Sheet functionality, and mainly it's looks on Both Android and iOS

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [x] If your changes are visual, did you check both the light and dark themes?
- [x] Added e2e tests? If not, please specify why
- [x] If you added new critical path files, did you update the CODEOWNERS file?
- [x] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: src/screens/RestoreSheet.js [hunk 1]
```diff
@@ -21,8 +20,9 @@ import { useDimensions } from '@/hooks';
 import { useNavigation } from '@/navigation';
 import Routes from '@/navigation/routesNames';
 import logger from '@/utils/logger';
+import { IS_ANDROID, IS_IOS } from '@/env';
 
-export default function RestoreSheet() {
+export function RestoreSheet() {
```
Comment (estrattonbailey) [inline src/screens/RestoreSheet.js:25]: 😍

**Review by estrattonbailey — APPROVED** [review]

**Review by skylarbarrera — APPROVED** [review]
hot asf

File: src/screens/RestoreSheet.js [hunk 2]
```diff
@@ -95,30 +95,34 @@ export default function RestoreSheet() {
     });
   }, [goBack, navigate]);
 
-  const wrapperHeight =
-    deviceHeight + longFormHeight + (android ? getSoftMenuBarHeight() / 2 : 0);
```
Comment (skylarbarrera) [inline src/screens/RestoreSheet.js:99]: @BrodyHughes we need your Samsung phone

Comment (jkadamczyk) [reply src/screens/RestoreSheet.js:99]: Screenshot 2022-10-21 at 14 20 08
It works pretty well

**Pull request merged**